src/generated/xkb.rs

```diff
@@ -7587,23 +7271,60 @@ where
         access_x_timeout_options_mask_bytes[1],
         access_x_timeout_options_values_bytes[0],
         access_x_timeout_options_values_bytes[1],
+        per_key_repeat[0],
+        per_key_repeat[1],
+        per_key_repeat[2],
+        per_key_repeat[3],
+        per_key_repeat[4],
+        per_key_repeat[5],
+        per_key_repeat[6],
+        per_key_repeat[7],
+        per_key_repeat[8],
+        per_key_repeat[9],
+        per_key_repeat[10],
+        per_key_repeat[11],
+        per_key_repeat[12],
+        per_key_repeat[13],
+        per_key_repeat[14],
+        per_key_repeat[15],
+        per_key_repeat[16],
+        per_key_repeat[17],
+        per_key_repeat[18],
+        per_key_repeat[19],
+        per_key_repeat[20],
+        per_key_repeat[21],
+        per_key_repeat[22],
+        per_key_repeat[23],
+        per_key_repeat[24],
+        per_key_repeat[25],
+        per_key_repeat[26],
+        per_key_repeat[27],
+        per_key_repeat[28],
+        per_key_repeat[29],
+        per_key_repeat[30],
+        per_key_repeat[31],
```

Comment:
This copy of `per_key_repeat` could be avoided. Dunno if it makes much of a difference or not.

Reply:
Fixed